explainaboard/processors/__init__.py

```diff
@@ -27,7 +27,7 @@
     text_pair_classification,
     word_segmentation,
 )
-from explainaboard.processors.processor_registry import get_processor
+from explainaboard.processors.processor_factory import get_processor
```

Comment:
Sorry for separate comments, but I realized that checking the behavior of `processor_factory` should be placed on a separate unit tests, and each unit test for a single processor should not import this module to avoid dependency on every other processor.

Reply:
Sorry this is commented on a wrong position: the comment above is for test code.

Reply:
Done in commit 50d7eba4a13b25adbd5bdcdef1c7d49767850976